fml/platform/android/jni_util.cc

```diff
@@ -26,7 +34,13 @@ void InitJavaVM(JavaVM* vm) {
 JNIEnv* AttachCurrentThread() {
   FML_DCHECK(g_jvm != nullptr)
       << "Trying to attach to current thread without calling InitJavaVM first.";
+
   JNIEnv* env = nullptr;
+  if (g_jvm->GetEnv(reinterpret_cast<void**>(&env), JNI_VERSION_1_4) ==
```

Comment:
nit: Can we document this behavior in the header file?

Reply:
done